test/hotspot/jtreg/compiler/c2/TestSubIdeal.java

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
You missed the copyright header here, the same for the microbenchmark

Reply:
Thank you for reviewing. This is my first pull request for OpenJDK. I was wondering what the format for the copyright should look like. I am a PhD student; should I state my university there? Thanks.

Reply:
I believe it depends on your OCA. If you signed it as an individual then the name appears there should be Oracle and/or its affiliates (read other files for the exact line), otherwise it would be the name of the company.
Cheers.